packages/client/src/generation/TSClient/Model.ts

```diff
@@ -330,29 +343,36 @@ export class ModelDelegate implements Generatable {
   ) {}
   public toTS(): string {
     const { fields, name } = this.outputType
-    const mapping = this.dmmf.mappingsMap[name]
-    if (!mapping) {
-      return ''
-    }
-    const model = this.dmmf.modelMap[name]
 
-    const actions = Object.entries(mapping).filter(
-      ([key, value]) => key !== 'model' && key !== 'plural' && key !== 'aggregate' && key !== 'groupBy' && value,
-    )
+    const mapping = this.dmmf.mappingsMap[name] ?? { model: name, plural: `${name}s` }
```

Comment:
Pluralizing is often a bad idea, we should remove it in the future (and stop adding more)